rca/schools/models.py

```diff
@@ -176,6 +178,13 @@ def __str__(self):
 
 
 class SchoolPage(ContactFieldsMixin, LegacyNewsAndEventsMixin, BasePage):
+    # Allow staff to see the new school pages for publishing, anyone else
+    # should receive a 404 and get redirected to the legacy site.
+    def serve(self, request):
+        if not request.user.is_staff:
+            raise Http404
+        return render(request, self.template, self.get_context(request))
```

Comment:
This is different to the Wagtail `Page` implementation, which takes into account previews, do you really want to override it? https://github.com/wagtail/wagtail/blob/master/wagtail/core/models.py#L1600

You could defer to default Wagtail behaviour with `return super().serve(request)`